changelog.txt

```diff
@@ -1,19 +1,36 @@
+4.23.0
+============
+
+New Features:
+**Windows Account Manager (WAM) is now available on .NET classic**. See issue [#2181](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2181) and [#2182](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2182) for details and https://aka.ms/msal-net-wam.
+**Proof of Possession (Signed HTTP Request) for confidential clients now support key management**. See issue [#2013](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2013) and ()[] for details.
+**MSAL .NET includes a fallback in case calling the local intance metadata service fails**. See issue [#2055](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2055) for details.
```

Comment:
```suggestion
**MSAL .NET includes a fallback in case calling the local instance metadata service fails due to unsupported version**. See issue [#2055](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2055) for details.
```

Reply:
thank you. updated